ReSharper.FSharp/src/FSharp.Psi.Features/src/Parsing/FSharpImplTreeBuilder.fs

```diff
@@ -216,15 +194,13 @@ type FSharpImplTreeBuilder(lexer, document, decls, lifetime, path, projectedOffs
     member x.ProcessTypeMember(typeMember: SynMemberDefn) =
         match typeMember with
         | SynMemberDefn.ImplicitCtor _ -> ()
+        | SynMemberDefn.LetBindings(bindings, _, _, range) ->
+            x.ProcessTopLevelBindings(bindings, range)
```

Comment:
Please update `typeBodyMemberDeclaration` interface in the grammar as well, as with this change it may not longer be a valid type member declaration.